content/en/docs/ambient/architecture/data-plane/index.md

```diff
@@ -46,8 +46,26 @@ requests directly to the destination without going through any waypoint proxy ev
 Currently, traffic from sidecars and gateways won't go through any waypoint proxy either and they will be made aware of waypoint proxies
 in a future release.
 
-#### Dataplane details
+#### L4 dataplane details
```

Comment:
I would prefer not to say "L4 dataplane".

Istio has a control plane and a data plane. How the data plane is implemented can be one of at least three ways.

Data plane for Layer 4 traffic?

First thought, not perfect.

It's wordier but may be easier to make consistent.

Reply:
fair 'nough.